### PR TITLE
Review messages in Directory Services.

### PR DESCRIPTION
**Pull request by Mrt134**

### CODE REVIEW

**Comment by erinix** [comment]
rebuild

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 1]
```diff
@@ -71,97 +71,97 @@ export class IdmapComponent implements OnInit {
...
-                this system is authoritative. Any lower UID/GID is\
-                ignored.'),
+                  this system is authoritative. Any lower UID/GID is\
+                  ignored.'),
     },
     {
       type: 'input',
       name: 'idmap_fruit_range_high',
       placeholder: T('Range High'),
       tooltip: T('Enter the integer of the ending UID/GID for which this\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: As above.

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 2]
```diff
@@ -71,97 +71,97 @@ export class IdmapComponent implements OnInit {
...
-                value is ignored.'),
+                  system is authoritative. Any UID/GID higher than this\
+                  value is ignored.'),
     }];
   public ldapFieldConfig: FieldConfig[] = [
     {
       type: 'input',
       name: 'idmap_ldap_range_low',
       placeholder: T('Range Low'),
       tooltip: T('Enter the integer of the beginning UID/GID for which\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: As above.

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 3]
```diff
@@ -71,97 +71,97 @@ export class IdmapComponent implements OnInit {
...
-                this system is authoritative. Any lower UID/GID is\
-                ignored.'),
+                  this system is authoritative. Any lower UID/GID is\
+                  ignored.'),
     },
     {
       type: 'input',
       name: 'idmap_ldap_range_high',
       placeholder: T('Range High'),
       tooltip: T('Enter the integer of the ending UID/GID for which this\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: As above.

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 4]
```diff
@@ -71,97 +71,97 @@ export class IdmapComponent implements OnInit {
...
     },
     {
       type: 'input',
       name: 'idmap_ldap_ldap_base_dn',
       placeholder: T('Base DN'),
       tooltip: T('Enter the directory base suffix to use for SID/uid/gid\
-                mapping entries. Example: dc=test,dc=org. When undefined\
-                <b>idmap_ldap</b> defaults to using the ldap idmap\
-                suffix option from <b>smb.conf</b>.'),
+                  mapping entries. Example: dc=test,dc=org. When undefined,\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: Needs markup around the `dc=test,dc=org`.  Preferably italics.

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 5]
```diff
@@ -71,97 +71,97 @@ export class IdmapComponent implements OnInit {
...
+                  idmap_ldap defaults to using the ldap idmap\
+                  suffix option from smb.conf.'),
     },
     {
       type: 'input',
       name: 'idmap_ldap_ldap_user_dn',
       placeholder: T('User DN'),
       tooltip: T('Enter the user Distinguished Name (DN) to use for\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter the user/User/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 6]
```diff
@@ -71,97 +71,97 @@ export class IdmapComponent implements OnInit {
...
-                authentication.'),
+                  authentication.'),
     },
     {
       type: 'input',
       name: 'idmap_ldap_ldap_url',
       placeholder: T('URL'),
       tooltip: T('Enter the LDAP server to use for SID/uid/gid map\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter the//

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 7]
```diff
@@ -197,98 +197,98 @@ export class IdmapComponent implements OnInit {
       name: 'idmap_nss_range_low',
       placeholder: T('Range Low'),
       tooltip: T('Enter the integer of the beginning UID/GID for which\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter the integer/Number/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 8]
```diff
@@ -197,98 +197,98 @@ export class IdmapComponent implements OnInit {
...
-                this system is authoritative. Any lower UID/GID is\
-                ignored.'),
+                  this system is authoritative. Any lower UID/GID is\
+                  ignored.'),
     },
     {
       type: 'input',
       name: 'idmap_nss_range_high',
       placeholder: T('Range High'),
       tooltip: T('Enter the integer of the ending UID/GID for which this\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter the integer/Number/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 9]
```diff
@@ -197,98 +197,98 @@ export class IdmapComponent implements OnInit {
...
-                value is ignored.'),
+                  system is authoritative. Any UID/GID higher than this\
+                  value is ignored.'),
     }];
   public rfcFieldConfig: FieldConfig[] = [
     {
       type: 'input',
       name: 'idmap_rfc2307_range_low',
       placeholder: T('Range Low'),
       tooltip: T('Enter the integer of the beginning UID/GID for which\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter the integer/Number/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 10]
```diff
@@ -197,98 +197,98 @@ export class IdmapComponent implements OnInit {
...
-                this system is authoritative. Any lower UID/GID is\
-                ignored.'),
+                  this system is authoritative. Any lower UID/GID is\
+                  ignored.'),
     },
     {
       type: 'input',
       name: 'idmap_rfc2307_range_high',
       placeholder: T('Range High'),
       tooltip: T('Enter the integer of the ending UID/GID for which this\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter the integer/Number/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 11]
```diff
@@ -197,98 +197,98 @@ export class IdmapComponent implements OnInit {
...
       placeholder: T('User Bind Path'),
       tooltip: T('Enter the search base where user objects can be\
-                found in the LDAP server.'),
+                  found in the LDAP server.'),
     },
     {
       type: 'input',
       name: 'idmap_rfc2307_bind_path_group',
       placeholder: T('Group Bind Path'),
       tooltip: T('Enter the search base where group objects can be\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter the/The/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 12]
```diff
@@ -197,98 +197,98 @@ export class IdmapComponent implements OnInit {
...
-                stand-alone LDAP server.'),
+                  LDAP server provided by the Active Directory server or a\
+                  stand-alone LDAP server.'),
       options: [],
     },
     {
       type: 'input',
       name: 'idmap_rfc2307_bind_path_user',
       placeholder: T('User Bind Path'),
       tooltip: T('Enter the search base where user objects can be\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter the/The/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 13]
```diff
@@ -197,98 +197,98 @@ export class IdmapComponent implements OnInit {
...
       placeholder: T('CN Realm'),
       tooltip: T('Append <i>@realm</i> to <i>cn</i> in LDAP queries for\
-                both groups and users when <b>User CN</b> is set).'),
+                  both groups and users when User CN is set).'),
     },
     {
       type: 'input',
       name: 'idmap_rfc2307_ldap_domain',
       placeholder: T('LDAP Domain'),
       tooltip: T('Enter the domain to access the Active Directory server\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter the/The/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 14]
```diff
@@ -197,98 +197,98 @@ export class IdmapComponent implements OnInit {
...
-                when using the LDAP server inside the Active Directory\
-                server.'),
+                  when using the LDAP server inside the Active Directory\
+                  server.'),
     },
     {
       type: 'input',
       name: 'idmap_rfc2307_ldap_url',
       placeholder: T('LDAP URL'),
       tooltip: T('Enter the LDAP URL for accessing the LDAP server when\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter the/The/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 15]
```diff
@@ -197,98 +197,98 @@ export class IdmapComponent implements OnInit {
...
-                using a stand-alone LDAP server.'),
+                  using a stand-alone LDAP server.'),
     },
     {
       type: 'input',
       name: 'idmap_rfc2307_ldap_user_dn',
       placeholder: T('LDAP User DN'),
       tooltip: T('Enter the user Distinguished Name to use for\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter the user/User/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 16]
```diff
@@ -197,98 +197,98 @@ export class IdmapComponent implements OnInit {
...
-                authentication.'),
+                  authentication.'),
     },
     {
       type: 'input',
       name: 'idmap_rfc2307_ldap_user_dn_password',
       placeholder: T('LDAP User DN Password'),
-      tooltip: T('Enter a password associated with the <b>LDAP User DN</b>.'),
+      tooltip: T('Enter a password associated with the LDAP User DN.'),
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter a password/Password/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 17]
```diff
@@ -330,84 +330,84 @@ export class IdmapComponent implements OnInit {
       name: 'idmap_rid_range_low',
       placeholder: T('Range Low'),
       tooltip: T('Enter the integer of the beginning UID/GID for which\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter the integer/Number/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 18]
```diff
@@ -330,84 +330,84 @@ export class IdmapComponent implements OnInit {
...
-                this system is authoritative. Any lower UID/GID is\
-                ignored.'),
+                  this system is authoritative. Any lower UID/GID is\
+                  ignored.'),
     },
     {
       type: 'input',
       name: 'idmap_rid_range_high',
       placeholder: T('Range High'),
       tooltip: T('Enter the integer of the ending UID/GID for which this\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter the integer/Number/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 19]
```diff
@@ -330,84 +330,84 @@ export class IdmapComponent implements OnInit {
...
-                value is ignored.'),
+                  system is authoritative. Any UID/GID higher than this\
+                  value is ignored.'),
     }];
   public scriptFieldConfig: FieldConfig[] = [
     {
       type: 'input',
       name: 'idmap_script_range_low',
       placeholder: T('Range Low'),
       tooltip: T('Enter the integer of the beginning UID/GID for which\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter the integer/Number/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 20]
```diff
@@ -330,84 +330,84 @@ export class IdmapComponent implements OnInit {
...
-                this system is authoritative. Any lower UID/GID is\
-                ignored.'),
+                  this system is authoritative. Any lower UID/GID is\
+                  ignored.'),
     },
     {
       type: 'input',
       name: 'idmap_script_range_high',
       placeholder: T('Range High'),
       tooltip: T('Enter the integer of the ending UID/GID for which this\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter the integer/Number/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 21]
```diff
@@ -330,84 +330,84 @@ export class IdmapComponent implements OnInit {
...
-                system is authoritative. Any UID/GID higher than this\
-                value is ignored.'),
+                  system is authoritative. Any UID/GID higher than this\
+                  value is ignored.'),
     },
     {
       type: 'input',
       name: 'idmap_script_script',
       placeholder: T('Script'),
       tooltip: T('Configure an external program to perform id mapping. See\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/id/ID/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 22]
```diff
@@ -330,84 +330,84 @@ export class IdmapComponent implements OnInit {
...
-                target="_blank">idmap_script(8)</a> for more details.'),
+                  <a href="http://samba.org.ru/samba/docs/man/manpages/idmap_script.8.html"\
+                  target="_blank">idmap_script(8)</a> for more details.'),
     }];
   public tdbFieldConfig: FieldConfig[] = [
     {
       type: 'input',
       name: 'idmap_tdb_range_low',
       placeholder: T('Range Low'),
       tooltip: T('Enter the integer of the beginning UID/GID for which\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter the integer/Number/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 23]
```diff
@@ -330,84 +330,84 @@ export class IdmapComponent implements OnInit {
...
-                this system is authoritative. Any lower UID/GID is\
-                ignored.'),
+                  this system is authoritative. Any lower UID/GID is\
+                  ignored.'),
     },
     {
       type: 'input',
       name: 'idmap_tdb_range_high',
       placeholder: T('Range High'),
       tooltip: T('Enter the integer of the ending UID/GID for which this\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter the integer/Number/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 24]
```diff
@@ -330,84 +330,84 @@ export class IdmapComponent implements OnInit {
...
-                value is ignored.'),
+                  system is authoritative. Any UID/GID higher than this\
+                  value is ignored.'),
     }];
   public tdb2FieldConfig: FieldConfig[] = [
     {
       type: 'input',
       name: 'idmap_tdb2_range_low',
       placeholder: T('Range Low'),
       tooltip: T('Enter the integer of the beginning UID/GID for which\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter the integer/Number/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 25]
```diff
@@ -330,84 +330,84 @@ export class IdmapComponent implements OnInit {
...
-                this system is authoritative. Any lower UID/GID is\
-                ignored.'),
+                  this system is authoritative. Any lower UID/GID is\
+                  ignored.'),
     },
     {
       type: 'input',
       name: 'idmap_tdb2_range_high',
       placeholder: T('Range High'),
       tooltip: T('Enter the integer of the ending UID/GID for which this\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/Enter the integer/Number/

File: src/app/pages/directoryservice/idmap/idmap.component.ts [hunk 26]
```diff
@@ -330,84 +330,84 @@ export class IdmapComponent implements OnInit {
...
-                system is authoritative. Any UID/GID higher than this\
-                value is ignored.'),
+                  system is authoritative. Any UID/GID higher than this\
+                  value is ignored.'),
     },
     {
       type: 'input',
       name: 'idmap_tdb2_script',
       placeholder: T('Script'),
       tooltip: T('Configure an external program for id mapping instead of\
```
Comment (wblock) [inline src/app/pages/directoryservice/idmap/idmap.component.ts]: s/id/ID/

File: src/app/pages/directoryservice/ldap/ldap.component.ts [hunk 27]
```diff
@@ -75,15 +75,15 @@ export class LdapComponent {
       name : 'ldap_basedn',
       placeholder : T('Base DN'),
       tooltip: T('Enter the top level of the LDAP directory tree to be\
```
Comment (wblock) [inline src/app/pages/directoryservice/ldap/ldap.component.ts]: s/Enter the top/Top/

File: src/app/pages/directoryservice/ldap/ldap.component.ts [hunk 28]
```diff
@@ -75,15 +75,15 @@ export class LdapComponent {
...
-                used when searching for resources. Example:\
-                <i>dc=test,dc=org</i>.')
+                  used when searching for resources. Example:\
+                  <i>dc=test,dc=org</i>.')
     },
     {
       type : 'input',
       name : 'ldap_binddn',
       placeholder : T('Bind DN'),
       tooltip: T('Enter the administrative account name on the LDAP server.\
```
Comment (wblock) [inline src/app/pages/directoryservice/ldap/ldap.component.ts]: s/Enter the administrative/Administrative/

File: src/app/pages/directoryservice/ldap/ldap.component.ts [hunk 29]
```diff
@@ -98,86 +98,86 @@ export class LdapComponent {
...
       placeholder : T('Allow Anonymous Binding'),
       tooltip: T('Set for the LDAP server to disable authentication and\
-                allow read and write access to any client.')
+                  allow read and write access to any client.')
     },
     {
       type : 'input',
       name : 'ldap_usersuffix',
       placeholder : T('User Suffix'),
       tooltip: T('Enter a suffix to add to a name when the user account is\
```
Comment (wblock) [inline src/app/pages/directoryservice/ldap/ldap.component.ts]: s/Enter a suffix/Suffix/

File: src/app/pages/directoryservice/ldap/ldap.component.ts [hunk 30]
```diff
@@ -98,86 +98,86 @@ export class LdapComponent {
...
-                added to the LDAP directory.')
+                  added to the LDAP directory.')
     },
     {
       type : 'input',
       name : 'ldap_groupsuffix',
       placeholder : T('Group Suffix'),
       tooltip: T('Enter a suffix to add to a name when the group is added\
```
Comment (wblock) [inline src/app/pages/directoryservice/ldap/ldap.component.ts]: s/Enter a suffix/Suffix/

File: src/app/pages/directoryservice/ldap/ldap.component.ts [hunk 31]
```diff
@@ -98,86 +98,86 @@ export class LdapComponent {
...
-                to the LDAP directory.')
+                  to the LDAP directory.')
     },
     {
       type : 'input',
       name : 'ldap_passwordsuffix',
       placeholder : T('Password Suffix'),
       tooltip: T('Enter a suffix to add to the password when it is added\
```
Comment (wblock) [inline src/app/pages/directoryservice/ldap/ldap.component.ts]: As above.

File: src/app/pages/directoryservice/ldap/ldap.component.ts [hunk 32]
```diff
@@ -98,86 +98,86 @@ export class LdapComponent {
...
-                to the LDAP directory.')
+                  to the LDAP directory.')
     },
     {
       type : 'input',
       name : 'ldap_machinesuffix',
       placeholder : T('Machine Suffix'),
       tooltip: T('Enter a suffix to add to the system name when it is\
```
Comment (wblock) [inline src/app/pages/directoryservice/ldap/ldap.component.ts]: As above.

File: src/app/pages/directoryservice/ldap/ldap.component.ts [hunk 33]
```diff
@@ -98,86 +98,86 @@ export class LdapComponent {
...
-                added to the LDAP directory.')
+                  added to the LDAP directory.')
     },
     {
       type : 'input',
       name : 'ldap_sudosuffix',
       placeholder : T('SUDO Suffix'),
       tooltip: T('Enter the suffix for LDAP-based users that need\
```
Comment (wblock) [inline src/app/pages/directoryservice/ldap/ldap.component.ts]: As above.

File: src/app/pages/directoryservice/ldap/ldap.component.ts [hunk 34]
```diff
@@ -98,86 +98,86 @@ export class LdapComponent {
...
-                <b>System/Certificates</b> menu.'),
+                  LDAP server CA must first be imported using the\
+                  System/Certificates menu.'),
       options : []
     },
     {
       type : 'input',
       name : 'ldap_timeout',
       placeholder : T('LDAP timeout'),
       tooltip: T('Add more time in seconds if a Kerberos ticket time out\
```
Comment (wblock) [inline src/app/pages/directoryservice/ldap/ldap.component.ts]: "LDAP timeout in seconds. Increase this value if a Kerberos ticket timeout occurs."

File: src/app/pages/directoryservice/ldap/ldap.component.ts [hunk 35]
```diff
@@ -190,31 +190,31 @@ export class LdapComponent {
       name : 'ldap_idmap_backend',
       placeholder : T('Idmap Backend'),
       tooltip: T('Select the backend used to map Windows security\
```
Comment (wblock) [inline src/app/pages/directoryservice/ldap/ldap.component.ts]: s/Select the backend/Backend/

File: src/app/pages/directoryservice/activedirectory/activedirectory.component.ts [hunk 36]
```diff
@@ -136,90 +136,90 @@ export class ActiveDirectoryComponent {
...
+                   target="_blank">domain/forest trusts</a> and managing\
+                   files on multiple domains is required. Setting will\
+                   generate more winbind traffic and slow down filtering\
+                   through user/group info.'),
     },
     {
       type : 'checkbox',
       name : 'ad_use_default_domain',
       placeholder : T('Use Default Domain'),
       tooltip : T('Unset to prepend the domain name to the username.\
```
[the remark this answers is not in the document]
Comment (Mrt134) [reply src/app/pages/directoryservice/activedirectory/activedirectory.component.ts]: Resolved, but GitHub isn't clearing this one for some reason.

**Comment by erinix** [comment]
rebuild

**Comment by erinix** [comment]
rebuild

**Pull request merged**